tox.ini

```diff
@@ -134,6 +134,7 @@ envlist =
 #      https://github.com/gevent/gevent/issues/903
     requests_gevent_contrib-{py36}-requests{208,209,210,211,212,213,219}-gevent{12,13}
     requests_gevent_contrib-py{37,38}-requests{208,209,210,211,212,213,219}-gevent13
+    sanic_contrib-py{37,38}-sanic
```

Comment:
Sanic supports Python 3.6, so we should be testing it here as well.

Reply:
Thanks for pointing this out Kyle. I think we are going to support Python 3.7 and 3.8; even though Sanic supports 3.6. cc @majorgreys .

Reply:
Is there any reason why we aren't going to support it?

Reply:
👋 Brett, I think the reason for not including Python 3.6 is that we can more easily support sanic after python 3.7 because of the addition of the ```contextvars``` module in 3.7; but we are still looking into this. 🙏 !